### PR TITLE
Jose

### PR DESCRIPTION
**Pull request by jcgzzc**

### CODE REVIEW

File: jose.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-s
-s
\ No newline at end of file
+i love waffles
```
Comment (jcgzzc) [inline jose.txt:1]: please capitalize `Waffles`

**Review by NickBlair21 — CHANGES_REQUESTED** [review]
Please add "i love eggs" so that we have a balanced breakfast (TM)

**Review by mellothomas-credera — APPROVED** [review]
Looks good to me! Thanks for maintaining a well-balanced diet!

File: jose.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
...
+i love bananas
+i love eggs
```
Comment (NickBlair21) [inline jose.txt:3]: Looks good!

**Review by NickBlair21 — APPROVED** [review]

**Pull request merged**